src/logic/safe/store/actions/__tests__/utils.test.ts

```diff
@@ -153,7 +153,7 @@ describe('extractRemoteSafeInfo', () => {
         FEATURES.ERC1155,
         FEATURES.SAFE_APPS,
         FEATURES.CONTRACT_INTERACTION,
-        SAFE_FEATURES.SAFE_TX_GAS_OPTIONAL,
+        FEATURES.SAFE_TX_GAS_OPTIONAL,
```

Comment:
I thought this was a safe-specific feature? Hence, it wasn't included in the [`FEATURES` enum](https://github.com/gnosis/safe-react-gateway-sdk/blob/0de0d8a2c6bdbfdd9b9d6470500c7bcbfac0eb30/src/types/chains.ts#L54) in the gateway types.

Reply:
That's correct but I see no harm in it being in this enum.

Reply:
Good point, reopened + added EIP1559.